javascript/javascript1/week1/homework_1.sql

```diff
@@ -0,0 +1,62 @@
+--1. Find out how many tasks are in the task table
+
+SELECT COUNT (*)
+FROM task
+
+--2. Find out how many tasks in the task table do not have a valid due date
+
+SELECT COUNT (*)
+FROM task 
+WHERE due_date IS NULL
+
+--3. Find all the tasks that are marked as done
+
+SELECT task.title, status.name
+FROM task JOIN status ON task.status_id = status.id
+WHERE status.name IN ('Done')
```

Comment:
As you are comparing with a single value, there is no need to check for the `IN` operator. It is enough (and more efficient) to just check for equality.